cluster/evaluator/launch_eval.py

```diff
@@ -28,6 +28,43 @@
 import time
 
 
+def cross_eval(m1_path=None, m2_path=None, jobname=None, completions=20):
+    """
+    black_path, white_path: full gs:// paths to the .pb files to match up
+    jobname: string, appended to the container, used to differentiate the job names
+    (e.g. 'minigo-gpu-evaluator-v5-123-v7-456')
+    """
+    if m1_path is None or m2_path is None or jobname is None:
+        print("call me better")
+        return
+    kubernetes.config.load_kube_config()
+    configuration = kubernetes.client.Configuration()
+    api_instance = kubernetes.client.BatchV1Api(
+        kubernetes.client.ApiClient(configuration))
+
+    raw_job_conf = open("cluster/evaluator/cc-evaluator.yaml").read()
+
+    os.environ['MODEL_BLACK'] = m1_path
+    os.environ['MODEL_WHITE'] = m2_path
+    os.environ['JOBNAME'] = jobname + 'bw'
+    env_job_conf = os.path.expandvars(raw_job_conf)
+
+    job_conf = yaml.load(env_job_conf)
+    job_conf['spec']['completions'] = completions
+
+    resp = api_instance.create_namespaced_job('default', body=job_conf)
+
+    os.environ['MODEL_WHITE'] = m1_path
+    os.environ['MODEL_BLACK'] = m2_path
+    os.environ['JOBNAME'] = jobname + 'wb'
```

Comment:
_wb

Reply:
Done.  (actually, k8s can't have underscores in the name so its a hyphen but close enough ;)